### ISSUE TITLE
Hyperparameter tuning and comparing metrics

### ISSUE THREAD

**Issue by vishaltechevangelist** [vishaltechevangelist/project-disease-variant-prediction-with-genomics#7]
As data scientist, I want to train the different classifier models and hyper tune it
- RandomForestClassifer
- SMOTE over sampling
- HistGradientBoosting
- LightGBM
- XGBoost

**Comment by vishaltechevangelist** [comment]
HistGradientBoosting

 precision    recall  f1-score   support

           0       0.55      0.17      0.26    253225
           1       0.61      0.91      0.73    387994
           2       0.75      0.60      0.67     92775
           3       0.04      0.03      0.03       103
           4       0.64      0.88      0.74       372
           5       0.04      0.10      0.06        92
           6       0.95      0.96      0.96      1877

    accuracy                           0.62    736438
   macro avg       0.51      0.52      0.49    736438
weighted avg       0.61      0.62      0.56    736438

**Comment by vishaltechevangelist** [comment]
XGBoost

precision    recall  f1-score   support

           0       0.57      0.47      0.52    253225
           1       0.68      0.80      0.74    387994
           2       0.80      0.62      0.70     92775
           3       0.75      0.03      0.06       103
           4       0.96      0.90      0.93       372
           5       0.61      0.22      0.32        92
           6       0.96      0.99      0.98      1877

    accuracy                           0.66    736438
   macro avg       0.76      0.58      0.61    736438
weighted avg       0.66      0.66      0.66    736438
